mypy/semanal_enum.py

```diff
@@ -13,6 +14,15 @@
 from mypy.semanal_shared import SemanticAnalyzerInterface
 from mypy.options import Options
 
+# Note: 'enum.EnumMeta' is deliberately excluded from this list. Classes that directly use
+# enum.EnumMeta do not necessarily automatically have the 'name' and 'value' attributes.
+ENUM_BASES: Final = frozenset((
+    'enum.Enum', 'enum.IntEnum', 'enum.Flag', 'enum.IntFlag',
+))
+ENUM_SPECIAL_PROPS: Final = frozenset((
+    'name', 'value', '_name_', '_value_', '_order_', '__order__',
```

Comment:
```suggestion
    'name', 'value', '_name_', '_value_', '_order_',
```
Only need it once :)

Reply:
But, `__order__` is a legacy version of `_order_` if I am correct. We still can use it like so:

```python
from enum import Enum

class E(Enum):
    __order__ = 'X Y Z'
    X = 1
    Y = 2
```

<img width="752" alt="Снимок экрана 2021-12-11 в 18 30 47" src="https://user-images.githubusercontent.com/4660275/145682162-e36d14b5-6cf9-453b-a026-92f8914875ad.png">

Or does mypy uses some magic to turn `__order__` into `_order_`?

Reply:
Sorry, I misread.